tests/core/editor/manual/detaching.html

```diff
@@ -0,0 +1,158 @@
+<style>
+	form ul {
+		list-style: none;
+	}
+	input[disabled]+label {
+		text-decoration: line-through;
+	}
+</style>
+
+<form id="controls" oninput="updateFormControlsDisableState()">
+	<section id="type">
+		<h2>Editor type:</h2>
+		<ul>
+			<li><input type="radio" name="editor" value="classic" checked><label>Classic</label></li>
+			<li><input type="radio" name="editor" value="inline"><label>Inline </label></li>
+			<li><input type="radio" name="editor" value="divarea"><label>Divarea</label></li>
+		</ul>
+	</section>
+	<section id="element">
+		<h2>Editor element:</h2>
+		<ul>
+			<li><input type="radio" name="element" value="textarea" checked><label>Textarea</label></li>
+			<li><input type="radio" name="element" value="div"><label>Div</label></li>
+		</ul>
+	</section>
+	<section id="destroy">
+		<h2>Destroy:</h2>
+		<ul>
+			<li><input type="radio" name="destroy" value="0" checked><label>synchronously</label></li>
+			<li><input type="radio" name="destroy" value="1"><label>async with 0 timeout</label></li>
+			<li><input type="radio" name="destroy" value="2"><label>before 'loaded' event</label></li>
+			<li><input type="radio" name="destroy" value="3"><label>before 'scriptLoader.load' callback called</label></li>
+			<li><input type="radio" name="destroy" value="4"><label>before iframe#onload</label></li>
```

Comment:
It looks like this button is disabled all the time despite using different options. Couldn't spot a way to enable it.

Reply:
It is enabled only when browser is Firefox or IE (not Edge), and editor type is `classic`. Reason for that is only in this case `iframe#onload` is used.

https://github.com/ckeditor/ckeditor-dev/blob/9cb74b35db7082189088f47326eb05b02c947f8d/plugins/wysiwygarea/plugin.js#L49-L53

Reply:
Do you see it readable in a current shape? It looks like there is something broken when testing not listed browsers, not like it has been disabled on purpose. I see 2 options:
* enable it despite browser sniffing, so it will prevent against regressions as browser sniffing should be treated as an internal mechanism
* write some note at this radio so it's known that it should only work for FF and IE

I'm rather for unifying test case among browsers (enabling it).

Reply:
I'll add an message in test. This condition is the same as one deciding if `iframe#onload` should be used. So it doesn't matter if browser sniffing is working or not. This case is enabled only in environment which uses this listener for creating editor. Also this case could break test in other browsers, so I'm for having it disabled when it shouldn't be tested.

https://github.com/ckeditor/ckeditor-dev/blob/334fef5a0da86a853b788a0d1e8201eb18624ec2/plugins/wysiwygarea/plugin.js#L51-L51